src/main/java/sg/ncl/webssh/SubscribeEventListener.java

```diff
@@ -0,0 +1,24 @@
+package sg.ncl.webssh;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.ApplicationListener;
+import org.springframework.messaging.simp.stomp.StompHeaderAccessor;
+import org.springframework.stereotype.Component;
+import org.springframework.web.socket.messaging.SessionSubscribeEvent;
+
+/**
+ * References:
+ * [1] http://www.devglan.com/spring-boot/spring-session-stomp-websocket
+ */
+
+@Component
+@Slf4j
+public class SubscribeEventListener implements ApplicationListener<SessionSubscribeEvent> {
+
+    @Override
+    public void onApplicationEvent(SessionSubscribeEvent sessionSubscribeEvent) {
+        StompHeaderAccessor headerAccessor = StompHeaderAccessor.wrap(sessionSubscribeEvent.getMessage());
+        log.info("onSubscribe [socket session: {}, spring session: {}]",
+                headerAccessor.getSessionId(), headerAccessor.getSessionAttributes().get("sessionId").toString());
```

Comment:
headerAccessor.getSessionAttributes has no key with "sessionId".

It contains only the following keys:
`org.springframework.messaging.simp.SimpAttributes.DESTRUCTION_CALLBACK.scopedTarget.webSocketBean=org.springframework.beans.factory.support.DisposableBeanAdapter@514ca317, scopedTarget.webSocketBean=sg.ncl.webssh.WebSocketBean@51bcf04d`

Reply:
Oops, forgot to remove this. This is supposed to keep the httpsession but no need now as ssh login should be independent from web service session.